ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -112,6 +112,90 @@ def test003_cancel_button_edit_departments(self, save):
                                                                                       order_departments))
             self.assertEqual(current_departments, order_departments)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test006_update_material_type(self, save):
+        """
+        New: Orders: Edit material type: Make sure that user able to change material type and related test plan &
+        article.
+
+        LIMS-4281
+        LIMS-4281
+        :return:
+        """
+        self.order_page.get_random_order()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        current_material_type = self.order_page.get_material_type()
+        self.order_page.set_material_type()
+        new_material_type = self.order_page.get_material_type()
+        self.order_page.confirm_popup(force=True)
+        self.order_page.get_article()
+        self.order_page.set_article()
+        self.order_page.get_test_plan()
+        self.order_page.set_test_plan()
```

Comment:
setting a general test_plan may be lead to `empty` article as u can choose material_type+article which doesn't have a test plan.
![image](https://user-images.githubusercontent.com/17981089/61034574-d2adc880-a3c5-11e9-94f6-48daf12ec57b.png)

Reply:
What about to give a test plan name in set_test_plan?? or get test plan only without set it

Reply:
if u change the Material type then you need to choose a related active article and test plan so I think u need to `set` it. As I said in the previouse comment,  So the solution is choosing a specific material type+active_article+test_plan to b valid